File: pinyougou-parent/pinyougou-shop-web/src/main/java/com/pinyougou/shop/controller/LoginController.java
package com.pinyougou.shop.controller;

import org.springframework.security.core.context.SecurityContextHolder;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

import java.util.HashMap;
import java.util.Map;
@RestController
@RequestMapping("/login")
public class LoginController {

    /**
     * 页面显示用户名
     * @return
     */
    @RequestMapping("/loginName")
    public Map findName(){
        String name = SecurityContextHolder.getContext().getAuthentication().getName();

        Map map = new HashMap();
        map.put("loginName",name);

        return map;
    }
}

File: pinyougou-parent/pinyougou-shop-web/src/main/webapp/js/controller/loginController.js
//控制层
app.controller('loginController' ,function($scope,$controller,loginService) {

//显示用户名
    $scope.findName = function () {
        loginService.findName().success(
            function (response) {
                $scope.loginName = response.loginName;
            }
        )
    }
})